ceremonies/2023/e5-cert.yaml

```diff
@@ -0,0 +1,27 @@
+ceremony-type: intermediate
+pkcs11:
+  module: /usr/lib/x86_64-linux-gnu/softhsm/libsofthsm2.so
+  pin: 1234
+  signing-key-slot: 1307844626
+  signing-key-label: root-x2
+inputs:
+  issuer-certificate-path: /run/shm/ceremonies/2020/root-x2.cert.pem
+  public-key-path: /run/shm/ceremonies/2023/int-e5.key.pem
+outputs:
+  certificate-path: /run/shm/ceremonies/2023/int-e5.cert.pem
+certificate-profile:
+  signature-algorithm: ECDSAWithSHA384
+  common-name: (FAKE) E5
+  organization: (FAKE) Let's Encrypt
+  country: XX
+  not-before: 2023-05-24 00:00:00
+  not-after: 2028-05-23 23:59:59
```

Comment:
I know we don't know the exact date that we'll end up doing the ceremony yet, but let's set these dates on the assumption that we'll be doing this at some point in September.